eip-0004.md

```diff
@@ -52,5 +52,6 @@ The standardization of various asset types can be found below:
 | --------------- |:-----------------------------------------------:| ----------------------------:|----------------------------:|
 | NFT - picture artwork              | [0x01, 0x01] - i.e., "0e020101"                        | SHA256 hash of the picture    | Optional - link to the artwork (UTF-8 representation) |
 | NFT - audio artwork              | [0x01, 0x02] - i.e., "0e020102"                        | SHA256 hash of the audio    | Optional - link to the audio (UTF-8 representation) |
+| Membership token - threshold signature              | [0x02, 0x01] - i.e., "0e020201"                        | Number of required signatures (Integer) - i.e., 4 in case of 4-of-10 threshold signature   | Deposit adrdress of the funds controlled by the threshold signature |
```

Comment:
Deposit adrdress => Deposit address

Reply:
Please specify type for R9 (SigmaProp ?)

Reply:
@kushti How about ergo tree byte array?

Reply:
@anon-real works as well